### PR TITLE
Chore/set request batch limit

### PR DESCRIPTION
**Pull request by xeon27**
# PR Type ([Feature | Fix | Documentation | Test])
Fix

## Short Description
Return bad request if batch size of request exceeds BATCH_REQUEST_SIZE value set in config.

## Tests Added
...

### CODE REVIEW

File: web/config.py [hunk 1]
```diff
@@ -30,3 +30,5 @@ class Config(object):
 
     CELERY_BROKER_URL = os.environ["CELERY_BROKER_URL"]
     CELERY_BACKEND_URL = os.environ["CELERY_BACKEND_URL"]
+
+    BATCH_REQUEST_LIMIT = 8
```
Comment (markcoatsworth) [inline web/config.py:34]: For consistency, we should put this value in the `.env` file and then read it in like the other config values.

**Review by markcoatsworth — CHANGES_REQUESTED** [review]
Added one tiny request but otherwise looks good.

File: web/config.py (hunk 1 above)
Comment (xeon27) [reply web/config.py]: Made this change.

Comment (markcoatsworth) [reply web/config.py]: Great thanks!

**Pull request merged**